test/zencode/zencode_exec.bats

```diff
@@ -195,3 +195,44 @@ EOF
 	save_output verified.json
 	assert_output '{"myMessage":"Dear Bob, your name is too short, goodbye - Alice.","output":["Zenroom_certifies_that_signatures_are_all_correct!"]}'
 }
+
+# TODO: how to test failure and error message with zencode_exec?
```

Comment:
zencode-exec returns Z->exitcode whose values are defined in zenroom.h
```c
// EXIT CODES
#define ERR_INIT 4
#define ERR_PARSE 3
#define ERR_EXEC 2
#define ERR_GENERIC 1 // EXIT_FAILURE
#define SUCCESS 0 // EXIT_SUCCESS
```

Reply:
At the end I was able to catch the message error! Bats seems not to save the output and error when using a `cat file | run executable`